assets/javascripts/kitten/helpers/utils/unique-id.test.js

```diff
@@ -0,0 +1,31 @@
+import { expect } from 'chai'
+import { uniqueIdUtils } from 'kitten/helpers/utils/unique-id'
+
+describe('getNextId()', () => {
+  it('get next id', () => {
+    const id = uniqueIdUtils.getNextId('k-MyComponent')
+    expect(id).to.be.equal('k-MyComponent-1')
+  })
+
+  it('get next id', () => {
```

Comment:
Ici le `it` a le même texte que le texte précédent, il faudrait mettre autre chose si on teste autre chose. Par exemple ici `it('increments the string when called twice', () => {`.